runtests.py

```diff
@@ -115,7 +115,7 @@ def get_distutils_distro(_cache=[]):
     distutils_distro = Distribution()
 
     if sys.platform == 'win32':
-        # TODO: Figure out why this hackery (see http://thread.gmane.org/gmane.comp.python.cython.devel/8280/).
+        # TODO: Figure out why this hackery (see https://thread.gmane.org/gmane.comp.python.cython.devel/8280/).
```

Comment:
This is actually a broken link now, but I guess we'll have to live with that. No idea what this was meant to refer to…

Reply:
All gmane web links are broken these days. I'd still keep them, just in case they ever come back. The service is still there, just the web interface isn't back (yet?).